examples/kubernetes/addons/etcd-operator/tls/certs/gen-cert.sh

```diff
@@ -3,12 +3,16 @@
 dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 if [ -z "$(which cfssl)" ]; then
-    echo "Please install the cfssl utility"
```

Comment:
Should we add this into the contributing guide as a dependency?

Reply:
I'd rather we put it into a docker image and have some type of wrapper script which generates stuff within that image so that we don't have to worry about maintaining more dependencies outside of images.

Reply:
This is a dependency on the user system.

* We probably don't want to require being able to run containers on the system that is running kubectl.
* There may be some resistance to running a random container image to generate certificates.

We can definitely offer it as an alternative but I would not make it the default.